.github/workflows/pull-request-token-metadata.yml

```diff
@@ -67,3 +67,26 @@ jobs:
 
       - name: Build and test example
         run: ./ci/cargo-test-sbf.sh token-metadata/example
+
+  js-test:
+    runs-on: ubuntu-latest
+    env:
+      NODE_VERSION: 16.x
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: node-${{ hashFiles('token-metadata/js/package-lock.json') }}
+          restore-keys: |
+            node-
+      - name: Download programs
+        uses: actions/download-artifact@v2
+        with:
+          name: token-lending-programs
+          path: target/deploy
```

Comment:
This is causing CI build problems.

Reply:
Removed, thanks for highlighting.

Hopefully fixes the issues - will need your help to trigger

Reply:
Looks like it worked!